faq.md

```diff
@@ -0,0 +1,375 @@
+
+    When backing up from a case-sensitive filesystem to a case-insensitive filesystem (such as Mac's HFS+ or Windows's FAT32 or NTFS), rdiff-backup escapes uppercase characters in filenames to make sure that no files are accidentally overwritten. When a filesystem is case-preserving but case-insensitive, it means that it remembers that a file is named "Foo" but doesn't distinguish between "Foo", "foo", "foO", "fOo", etc. However, filesystems such as Linux's ext3 do treat these names as separate files.
+
+    Imagine you have a Linux directory with two files, "bar" and "BAR", and you copy them to a Mac system. You will wind up with only one file (!) since HFS+ doesn't distinguish between the names, and the second file copied will overwrite the first. Therefore, when rdiff-backup copies files from case-sensitive to case-insensitive filesystems, it escapes the uppercase characters (eg, "M" is replaced with ";077", and "F" with ";070") so that no filename conflicts occur. Upon restore (from the Mac backup server to the Linux system), the filenames are unquoted and you will get "MyFile" back.
+
+8.  **<a name="remove_dir">My backup set contains some files that I just realized I don't want/need backed up. How do I remove them from the backup volume to save space?</a>**
+
+    The only official way to remove files from an rdiff-backup repository is by letting them expire using the --remove-older-than option. Deleting increments from the rdiff-backup-data directory will prevent you from recovering those files, but shouldn't prevent the rest of the repository from being restored.
+
+9.  **<a name="solaris">Does rdiff-backup work under Solaris?</a>**
+
+    There may be a problem with rdiff-backup and Solaris' libthread. Adding "ulimit -n unlimited" may fix the problem though. Here is a post by Kevin Spicer on the subject:
+
+    <pre>Subject: RE: Crash report....still not^H^H^H working
+    From: "Spicer, Kevin" <kevin.spicer@bmrb.co.uk>
+    Date: Sat, 11 May 2002 23:36:42 +0100
+    To: rdiff-backup@keywest.Stanford.EDU
+
+    Quick mail to follow up on this..
+    My rdiff backup (on Solaris 2.6 if you remember) has now worked
+    reliably for nearly two weeks after I added...
+
+        ulimit -n unlimited
+
+    to the start of my cron job and created a wrapper script on the remote
+    machine which looked like this...
+
+        ulimit -n unlimited
+        rdiff-backup --server
+        exit
+
+    And changed the remote schema on the command line of rdiff-backup to
+    call the wrapper script rather than rdiff-backup itself on the remote
+    machine.  As for the /dev/zero thing I've done a bit of Googleing and
+    it seems that /dev/zero is used internally by libthread on Solaris
+    (which doesn't really explain why its opening more than 64 files - but
+    at least I think I've now got round it).
+    </pre>
+
+10.  **<a name="speed">How fast is rdiff-backup? Can it be run on large data sets?</a>**
```

Comment:
This point and all following points are rendering as "1", might need to clean up the syntax.